test/machines/machines_test.go

```diff
@@ -191,7 +191,13 @@ var _ = framework.SigKubeDescribe("Machines", func() {
 			f.CreateClusterAndWait(cluster)
 
 			// Create master machine and verify the master node is ready
-			masterUserDataSecret := manifests.MasterMachineUserDataSecret("masteruserdatasecret", testNamespace.Name)
+			masterUserDataSecret, err := manifests.MasterMachineUserDataSecret(
+				"masteruserdatasecret",
+				testNamespace.Name,
+				[]string{"\\$(curl -s http://169.254.169.254/latest/meta-data/public-hostname)", "\\$(curl -s http://169.254.169.254/latest/meta-data/public-ipv4)"},
```

Comment:
Why are we escaping here? Would a string `literal` suffice?

Reply:
It's used inside a template